modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -1,83 +1,16 @@
 {{#models}}
 {{#model}}
-#import "SWGDate.h"
 #import "{{classname}}.h"
 
 @implementation {{classname}}
-
--(id){{#vars}}{{name}}: ({{datatype}}) {{name}}
-    {{/vars}}
-{{newline}}{
-    {{#vars}}_{{name}} = {{name}};
-    {{/vars}}
-
-    return self;
-}
-
--(id) initWithValues:(NSDictionary*)dict
+{{#vars}}
```

Comment:
I don't know how this can work.  Declaring keyMapper in the `{{#vars}}` loop will cause duplicate declarations, like this from the petstore example:

```objc
#import "SWGCategory.h"

@implementation SWGCategory
    
+ (JSONKeyMapper *)keyMapper
{
    return [[JSONKeyMapper alloc] initWithDictionary:@{@"id": @"_id"}];    
}
    
    // compilation error
+ (JSONKeyMapper *)keyMapper
{
    return [[JSONKeyMapper alloc] initWithDictionary:@{@"name": @"name"}];    
}
    
@end

```